ports/gsasl/vcpkg.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "gsasl",
+  "version": "2.2.1",
+  "description": "Simple Authentication and Security Layer framework and a few common SASL mechanisms",
+  "homepage": "https://www.gnu.org/software/gsasl/",
+  "license": "GPL-3.0-or-later",
+  "supports": "!uwp",
+  "dependencies": [
+    {
+      "name": "gettext",
+      "platform": "windows | android"
+    }
+  ]
```

```suggestion
  "supports": "!uwp"
```
(Would need elaboration for target and host dependencies.)